From 03cb1e3a8971df03e1df10693929fa5966583dfc Mon Sep 17 00:00:00 2001
From: Waldemar Rymarkiewicz <waldemarx.rymarkiewicz@intel.com>
Date: Fri, 6 Jul 2018 11:03:25 +0200
Subject: [PATCH] PMCPESYS-48: Use common name for syscon property in thermal

This commit updates the name of the syscon property used in thermal sensor
to the one that is used commonly in Intel drivers.

Signed-off-by: Waldemar Rymarkiewicz <waldemarx.rymarkiewicz@intel.com>
---
 Documentation/devicetree/bindings/thermal/ltq-thermal.txt | 4 ++--
 drivers/thermal/ltq_thermal.c                             | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/thermal/ltq-thermal.txt b/Documentation/devicetree/bindings/thermal/ltq-thermal.txt
index 3352de783947..0c95a1d53151 100644
--- a/Documentation/devicetree/bindings/thermal/ltq-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/ltq-thermal.txt
@@ -8,7 +8,7 @@ Required properties:
 - compatible:
   - "lantiq,ts-xrx500" for Lantiq GRX500 SoC
 
-- lantiq,chiptop: phandle to syscon node
+- intel,syscon: phandle to syscon node
 
 
 Example of sensor definition:
@@ -17,7 +17,7 @@ ts: ts@080000 {
 	#thermal-sensor-cells = <0>;
 
 	compatible = "lantiq,ts-xrx500";
-	lantiq,chiptop = <&sysconf>;
+	intel,syscon = <&sysconf>;
 };
 
 
diff --git a/drivers/thermal/ltq_thermal.c b/drivers/thermal/ltq_thermal.c
index c8b7e459518f..e102e5d5927c 100644
--- a/drivers/thermal/ltq_thermal.c
+++ b/drivers/thermal/ltq_thermal.c
@@ -247,9 +247,9 @@ static int ltq_thermal_probe(struct platform_device *pdev)
 	priv->dev = &pdev->dev;
 	platform_set_drvdata(pdev, priv);
 
-	priv->chiptop = syscon_regmap_lookup_by_phandle(node, "lantiq,chiptop");
+	priv->chiptop = syscon_regmap_lookup_by_phandle(node, "intel,syscon");
 	if (IS_ERR(priv->chiptop)) {
-		dev_err(&pdev->dev, "Failed to find lantiq,chiptop regmap\n");
+		dev_err(&pdev->dev, "Failed to find intel,syscon regmap\n");
 		return PTR_ERR(priv->chiptop);
 	}
 
